src/utils/errorReporting.ts

```diff
@@ -24,31 +23,16 @@ const reportError = (err: CustomBugsnagError): reportErrorResult => {
 
   return {
     severe: () => {
-      sendError(err, "error");
+      bugsnagSendError(err, "error");
+      sentrySendError(err, "error");
     },
     warning: () => {
-      sendError(err, "warning");
+      bugsnagSendError(err, "warning");
+      sentrySendError(err, "warning");
     },
   };
 };
 
-const sendError = (err: CustomBugsnagError, severity: Event["severity"]) => {
-  const userId = localStorage.getItem("userId");
-  let metadata;
-  if (err.metadata) {
-    metadata = err.metadata;
-  }
-  Bugsnag.notify(err, (event) => {
-    // reassigning param is recommended usage in bugsnag docs
-    // eslint-disable-next-line no-param-reassign
-    event.severity = severity;
-    event.setUser(userId);
-    if (metadata) {
-      event.addMetadata("metadata", { ...metadata });
-    }
-  });
-};
-
 const leaveBreadcrumb = (
```

Comment:
`leaveBreadcrumb` doesn't appear to be updated for sentry support. This is important because it notifies us of important events in our code such as which graphql requests were performed and what variables were used. Its also used pretty extensively on parsley to log important functions.

Reply:
Yeah I mentioned this in the description, the scope of this felt pretty major and I didn't want to bloat it too much or delay any work from being reviewed while I'm OOO. There's a separate ticket now. Would you maybe want to take it on as an immediate followup?